schema/fs.go

```diff
@@ -169,6 +169,7 @@ func _escFSByte(useLocal bool, name string) ([]byte, error) {
 		if err != nil {
 			return nil, err
 		}
+		defer f.Close()
```

Comment:
This is a generated file (`make schema-fs`), per https://github.com/mjibson/esc/pull/27#issuecomment-226769594